orchestration/dagster_orchestration/hca_orchestration/pipelines/load_hca.py

```diff
@@ -61,5 +62,6 @@
 @pipeline(
     mode_defs=[prod_mode, dev_mode, local_mode, test_mode]
 )
-def stage_data() -> None:
-    create_staging_dataset(pre_process_metadata(clear_staging_dir()))
+def load_hca() -> None:
+    staging_dataset: HcaStagingDatasetName = create_staging_dataset(pre_process_metadata(clear_staging_dir()))
```

Comment:
this type annotation should be redundant - it'll be enforced by the input schema for `import_data_files`